community/planning/biome_oauth_user_store.md

```diff
@@ -0,0 +1,205 @@
+# Biome OAuth Integration
```

Comment:
Can you update the OAuth design doc to link to this one, and remove the unanswered question that this doc answers?

Reply:
Added